ihaskell-display/ihaskell-widgets/src/IHaskell/Display/Widgets.hs

```diff
@@ -77,4 +77,6 @@ import           IHaskell.Display.Widgets.Common as X
 import           IHaskell.Display.Widgets.Types as X (setField, getField, properties, triggerDisplay,
                                                       triggerChange, triggerClick, triggerSelection,
                                                       triggerSubmit, ChildWidget(..), StyleWidget(..),
-                                                      WidgetFieldPair(..), Date(..), unlink)
+                                                      WidgetFieldPair(..), Date(..), unlink,
+                                                      JSONByteString(..), OutputMsg(..))
```

Comment:
The conflict comes because I forgot to add the JSONByteString constructor to the export list. It's here, though, so it should be safe to just accept this file and ignore mine.

Reply:
@jamesdbrock can you rebase?

Reply:
Rebased.